src/react-apps/applications/shared/__tests__/appBar.test.tsx

```diff
@@ -0,0 +1,134 @@
+import { createMuiTheme, MuiThemeProvider } from '@material-ui/core/styles';
+import { mount } from 'enzyme';
+import * as React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import * as renderer from 'react-test-renderer';
+import AppBarComponent from '../src/navigation/main-header/appBar';
+import altinnTheme from '../src/theme/altinnStudioTheme';
+
+describe('AppBarComponent - src/navigation/main-header/appBar - snapshot...', () => {
+  let mockOrg: string;
+  let mockService: string;
+  let mockActiveSubHeaderSelection: string;
+  let mockShowSubheader: boolean;
+
+  beforeEach(() => {
+    mockOrg = 'jest-test-org';
+    mockService = 'jest-test-service';
+    mockActiveSubHeaderSelection = 'create';
+    mockShowSubheader = true;
+
+  });
+
+  it('should match snapshot with default values', () => {
+    const rendered = renderer.create(
+      <MemoryRouter>
+        <AppBarComponent
+          org={mockOrg}
+          service={mockService}
+          showSubHeader={mockShowSubheader}
+          activeSubHeaderSelection={mockActiveSubHeaderSelection}
+        />
+      </MemoryRouter>,
+    );
+    expect(rendered).toMatchSnapshot();
+  });
+
+  it('should match snapshot with subHeader and Publish selection active', () => {
+    const rendered = renderer.create(
+      <MemoryRouter>
+        <AppBarComponent
+          org='other-org'
```

Comment:
probably these values can also be replaced with variables to look more clear and consistent with other tests

Reply:
Fixed.